packages/drivers/tinylicious-driver/package.json

```diff
@@ -32,6 +32,7 @@
     "@fluidframework/driver-utils": "^0.40.0",
     "@fluidframework/protocol-definitions": "^0.1025.0-0",
     "@fluidframework/routerlicious-driver": "^0.40.0",
+    "@fluidframework/server-services-client": "^0.1025.0-0",
```

Comment:
Instead of taking this dependency on a service package, should there be a client package that also provides the getRandomName function?

Reply:
That could be in common-utils, no? We need to make sure whatever we use meets internationalization requirements etc.

Reply:
Even if it builds with service, I think clients use it extensively (at least driver does).

Reply:
@tanviraumi Yes, routerlicious-driver already has multiple references to it. @tylerbutler should I continue to have it reference this package then since it seems its expected that this package is shared across service and client

Reply:
even if you want to move it to common-utils the way prerelease packages are published will prevent you from making the change here anyway 🙃

Reply:
@sumedhb1995 Leave it as-is. Shouldn't block this change on it.

Reply:
@heliocliu Yep, I was wondering how I was going to reference 1.026 that would contain these changes and looks like the answer is to wait for them to publish 😄